tools/amd_build/build_amd.py

```diff
@@ -201,6 +201,19 @@ def remove_hcc(line: str) -> str:
                     sources.write(line)
             print(f"{hip_platform_file} updated")
 
+# NOTE: fbgemm sources needing hipify
```

Comment:
I think it's a very good note and kind of indicative of the problem under the hood: 
If one to want to use those files from torch_cuda, fbgemm should export a proper target and torch should depend on it.

Reply:
@cthi for viz